sdk/storage/Azure.Storage.DataMovement.Blobs/README.md

```diff
@@ -40,7 +40,178 @@ az storage account create --name MyStorageAccount --resource-group MyResourceGro
 ### Authenticate the client
 In order to interact with the Data Movement library you have to create an instance with the TransferManager class.
 
-TODO: Insert code snippet here.
+```C# Snippet:UploadSingle_ConnectionStringAsync
+    // Create simple transfer single blob upload job
+    DataTransfer dataTransfer = await transferManager.StartTransferAsync(
+        sourceResource: new LocalFileStorageResource(sourceLocalPath),
+        destinationResource: new BlockBlobStorageResource(destinationBlob));
+    await dataTransfer.AwaitCompletion();
+}
+```
+
+```C# Snippet:DownloadSingle_SharedKeyAuthAsync
+    // Simple Download Single Blob Job
+    StorageResource sourceResource = new BlockBlobStorageResource(sourceBlob);
+    StorageResource destinationResource = new LocalFileStorageResource(downloadPath);
+    await transferManager.StartTransferAsync(
+        sourceResource,
+        destinationResource);
+
+    StorageResource sourceResource2 = new BlockBlobStorageResource(sourceBlob);
+    StorageResource destinationResource2 = new LocalFileStorageResource(downloadPath2);
+
+    await transferManager.StartTransferAsync(
+        sourceResource: new BlockBlobStorageResource(sourceBlob, new BlockBlobStorageResourceOptions()
+            {
+                DownloadOptions = new BlobStorageResourceDownloadOptions()
+                {
+                    Conditions = new BlobRequestConditions(){ LeaseId = "xyz" }
+                }
+            }),
+        destinationResource: new LocalFileStorageResource(downloadPath2));
```

Comment:
I think we can separate this sample out as how to use apply options on the `BlockBlobStorageResource` when downloading.